lib/scr/screen_dice_roller.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:simple_dice_roller/utils/constants_file.dart';
+
+class ScreenDiceRoller extends StatefulWidget {
+  final String title;
+  final int numberOfDice;
+
+  ScreenDiceRoller({
+    Key key,
+    this.title = "Dice Roller App",
+    this.numberOfDice = Constants.numberOfDice,
+  }) : super(key: key);
+
+  @override
+  _ScreenDiceRollerState createState() => _ScreenDiceRollerState();
+}
+
+class _ScreenDiceRollerState extends State<ScreenDiceRoller> {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: widget.title,
+      home: Scaffold(
+        backgroundColor: Colors.cyan.shade700,
+        appBar: AppBar(
+          title: Text(
+            widget.title,
+            style: TextStyle(
+              color: Colors.cyan,
+              fontWeight: FontWeight.bold,
+              fontSize: Constants.fontSize_title,
+            ),
+          ),
+          backgroundColor: Colors.black,
+        ),
+        body: GridView.count(
+          crossAxisCount: 2,
+          children: List.generate(widget.numberOfDice, (index) {
+            return Center(
+              child: Image.asset(
+                "assets/images/Dice_${(index % Constants.mob_number) + 1}.png",
+                height: Constants.image_height,
+              ),
+            );
+          }),
+        ),
+        bottomNavigationBar: Container(
+          padding: EdgeInsets.fromLTRB(0, 0, 0, 70),
```

Comment:
instead of using the padding inside of the Container, you can wrap the widget with a Padding.
As you only change the bottom padding, you should use, EdgeInsets.only(bottom: 70), and the 70 should be a constant.

Reply:
Done